tests/test_policy.py

```diff
@@ -315,50 +317,56 @@ def test_config_resource_support(self):
         # of a resource.
 
         whitelist = {
-            'AWS::Kinesis::StreamConsumer',
-            'AWS::CodeDeploy::DeploymentConfig',
-            'AWS::OpenSearch::Domain',  # this is effectively an alias
+            'AWS::ApiGatewayV2::Api',
```

Comment:
this is a sorted list of the left + some new entries (Launch Template, EMR, SageMaker, Guardduty entires)